packages/cli/src/commands/models.ts

```
@@ -0,0 +1,79 @@
+import { listPiAIModels, listPiAIProviders } from "@obora/adapters";
+import { Command } from "commander";
+
+import { handleCommandAction } from "../utils/error-handler.js";
+import { formatter } from "../utils/formatter.js";
+import { getGlobalOpts } from "../utils/global-opts.js";
+
+interface ModelsOptions {
+  json?: boolean;
+}
+
+export async function runModels(provider: string | undefined, options: ModelsOptions): Promise<void> {
```

ESLint

[error] Replace `provider:·string·|·undefined,·options:·ModelsOptions` with `⏎··provider:·string·|·undefined,⏎··options:·ModelsOptions⏎`

(prettier/prettier)